Index.html

```diff
@@ -0,0 +1,200 @@
+    <link rel="stylesheet" href="./css/style.css"/>
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
+    <title>Imaginary Dance</title>
+  </head>
+  <body>
+    <div class="main">
+      <header>
+        <div class="top-header">
+          <ul class="top-header-list">
+            <li>
+              <a href=""><i class="fab fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fab fa-twitter"></i></a>
+            </li>
+            <li><a href="">English</a></li>
+            <li><a href="Index.html">My Page</a></li>
+            <li><a href="">Logout</a></li>
+          </ul>
+        </div>
+        <nav>
+          <div class="modal">
+            <img src="./Image/Enabled.svg" class="close" alt="closeTag" />
+            <div class="modal-body">
+              <ul class="modal-menu">
+                <li><a href="index.html">Home</a></li>
+                <li><a href="About.html">About</a></li>
+                <li><a href="#">Feature</a></li>
+              </ul>
+            </div>
+          </div>
+          <div class="logo">
+            <img src="./image/logo.png" class="logo-main" alt="logo" />
+          </div>
+          <div class="nav-hamburger">
+            <img src="./image/button.png" class="hamburger" alt="menu-bar" />
+          </div>
+          <div class="menu-icons">
+            <ul>
+              <li><a href="index.html">Home</a></li>
+              <li><a href="#">Feature</a></li>
+              <li><a href="about.html">About</a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+      <section class="home" id="home-bg">
+        <div class="home-container">
+          <div class="dance-logo">
+            <img
+              src="https://img.icons8.com/ios-filled/100/000000/ballroom-dance.png"
+              alt="dancing-logo"
+            />
+          </div>
+          <div class="title">
+            <h1>"The Imaginary Dance"</h1>
+            <h2>Bringing life through dance</h2>
+          </div>
+          <div class="home-info">
+            <p>
+              “Stars are always dancing. Sometimes they dance twinkling away
+              with the rhythm of your joyful heart and sometimes they dance
+              without movement to embrace your heartache as if frozen sculptures
+              of open-armed sadness.”
+            </p>
+          </div>
+          <div class="home-info-2">
+            <p>
+              This is why the idea of finding love across the dance floor
+              endures – symbolizing that, when we know the true rhythm of our
+              heart, we know the other.
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="home" id="home2">
+        <div class="program-title">
+          <h2>Dance Guide</h2>
+        </div>
+        <div class="program-items">
```

Comment:
- [ ] Each `Card` is missing an `orange` title on both `mobile` and `desktop` views.
- [ ] Please make sure to add the missing section divider `-`, a title for each card, and make sure that individual articles have their elements organized in a vertical column, with the image at the top, title in the center, and description at the bottom on `desktop`.

Reference Design
![image](https://user-images.githubusercontent.com/33767951/142338247-d15ee6d9-ec35-4529-bea9-c2c8dab296dc.png)
Your Styling `Home => 786px`
![image](https://user-images.githubusercontent.com/33767951/142338353-526e6193-798b-48a8-816e-f3f0234e39fb.png)